tests/com/jvm/external/UnsafeSharedCacheTest.java

```diff
@@ -1,54 +0,0 @@
-package com.jvm.external;
-
-import static org.hamcrest.core.StringContains.containsString;
-import static org.junit.Assert.assertThat;
-import static org.junit.Assert.assertTrue;
-
-import java.io.IOException;
-import java.net.URI;
-import java.net.URISyntaxException;
-import java.net.URL;
-import java.net.URLClassLoader;
-import java.nio.file.Files;
-import java.nio.file.Path;
-import java.nio.file.Paths;
-import org.junit.Test;
-
-public class UnsafeSharedCacheTest {
-
-  private static String OS = System.getProperty("os.name").toLowerCase();
-
-  @Test
-  public void test_jarsOnClassPath_areInTheSharedCache() throws IOException, URISyntaxException {
-    // No support for symlinks on Windows. Let's skip that.
-    if (OS.indexOf("win") != -1) {
-      return;
-    }
-    boolean foundGuavaJar = false;
-    ClassLoader cl = ClassLoader.getSystemClassLoader();
-    for (URL url : ((URLClassLoader) cl).getURLs()) {
-      URI uri = url.toURI();
-      String uriString = uri.toString();
-      if (uriString.contains("repo1.maven.org") && uriString.contains("guava")) {
-        foundGuavaJar = true;
-        if (OS.indexOf("mac") >= 0) {
```

Comment:
Does this test not pass on Mac?  Or do we just not need this test anymore?

Reply:
It doesn't pass anywhere. Ever since we switched to adding explicit visibilities on targets rather than having it set at the package level (#649), this test hasn't passed anywhere since we're not exporting the files. I've no idea why our CI runs haven't caught this, but it's been that way for over a year.

Reply:
Let's delete the feature, then?

Reply:
Sure. I can do that.